assets/js/modules/tagmanager/components/common/FormInstructions.js

```diff
@@ -113,6 +112,21 @@ export default function FormInstructions() {
 		);
 	}
 
+	if ( hasExistingTag ) {
+		return (
+			<p>
+				{ sprintf(
+					// translators: %s: the existing container ID.
+					__(
+						'An existing tag was found on your site (%s). If you later decide to replace this tag, Site Kit can place the new tag for you. Make sure you remove the old tag first. Please select your Tag Manager account and container below, the snippet will be inserted automatically on your site.',
```

Comment:
Instead of concatenating the two texts here, let's keep using the two original translation strings without changes, which will ensure that translators don't have to provide new translations for them unnecessarily.

I'd say we can probably even render the two `<p>` tags next to each other, the two sentences don't have to be in the same paragraph. So maybe integrate the `hasExistingTag` condition here into the default return value below.